infra/containerregistry.bicep

```diff
@@ -0,0 +1,27 @@
+param location string
+param nameseed string
+param tags object = {}
+
+resource containerRegistry 'Microsoft.ContainerRegistry/registries@2022-02-01-preview' = {
+  name: 'cr${nameseed}${uniqueString(resourceGroup().id, location)}'
```

Comment:
Could you explain further why you have deviated from using resourceToken?  It seems to effectively do the same thing and we'd like to be consistent across samples that are published as canonical samples, so unless there is a REALLY good reason for deviating I would like to remain consistent.

Reply:
Resource token use in templates seems to be used to name all resources uniformly to comply with some services requirement to be globally unique.
I don't like it as the primary part of the name, nor its use in services that don't need to be unique.

For the vault and Registry I can pass the resource token to provide the unique element.... but I'm not sure if that would satisfy your point.

Reply:
1. resourceToken does get around the problem of globally unique names, but many Azure resources don't require globally unique names - in these cases it seems to be somewhat of a blunt instrument
1. resourceToken does not result in a meaningful azure resource name for the workload/app. I much prefer the format of ${consistent abbreviation}${relevant name}${optional unique suffix} similar to what you'd see [here](https://docs.microsoft.com/azure/cloud-adoption-framework/ready/azure-best-practices/resource-naming)

I've removed my naming approach and fallen back to the resourceToken approach for sake of sample consistency 😄